wasm/jit/jit_amd64_test.go

```diff
@@ -1147,26 +1147,26 @@ func TestAmd64Builder_handleLe(t *testing.T) {
 			)
 			// Check the stack.
 			require.Equal(t, uint64(1), eng.stackPointer)
-			if tc.exp {
-				require.Equal(t, uint64(1), eng.stack[eng.stackPointer-1])
+			if tc.signed {
+				require.Equal(t, int32(tc.x1) <= int32(tc.x2), eng.stack[eng.stackPointer-1] == 1)
 			} else {
-				require.Equal(t, uint64(0), eng.stack[eng.stackPointer-1])
+				require.Equal(t, uint32(tc.x1) <= uint32(tc.x2), eng.stack[eng.stackPointer-1] == 1)
 			}
 		}
 	})
 	t.Run("int64", func(t *testing.T) {
 		for _, tc := range []struct {
-			x1, x2      int64
-			signed, exp bool
+			x1, x2 int64
+			signed bool
 		}{
-			{x1: 100, x2: -1, signed: false, exp: true},
-			{x1: -1, x2: 100, signed: false, exp: false},
-			{x1: 100, x2: 200, signed: true, exp: true},
-			{x1: 200, x2: 100, signed: true, exp: false},
-			{x1: 1 << 56, x2: 100, signed: true, exp: false},
-			{x1: 1 << 56, x2: 1 << 61, signed: true, exp: true},
-			{x1: math.MaxInt64, x2: 100, signed: true, exp: false},
-			{x1: math.MinInt64, x2: 100, signed: true, exp: true},
+			{x1: 100, x2: -1, signed: false},
```

Comment:
a comment to help explain why we have signed false here

```suggestion
			{x1: 100, x2: -1, signed: false}, // interpret x2 as max uint64
```